src/bvh/bvh_impl.rs

```diff
@@ -377,7 +377,9 @@ impl<T: BHValue, const D: usize> Bvh<T, D> {
         } = self.nodes[node_index]
         {
             let joint_aabb = child_l_aabb.join(&child_r_aabb);
-            assert!(joint_aabb.relative_eq(outer_aabb, T::epsilon()));
+            // Avoid `relative_eq`, since there is no reason for imprecision, since
+            // Aabb's are joined by selecting the min/max of coordinates.
```

Comment:
The comment doesn't really make sense without the context of this change (the reader won't know it was asserting with `relative_eq` before) I think. Perhaps instead explain why it's correct to do an exact comparison.